orographic_precipitation/fastscape_ext.py

```diff
@@ -57,4 +60,29 @@ def run_step(self):
                                                 self.dx,
                                                 self.dy,
                                                 **self._params)
-        self.precip_rate = self.precip_rate[:] * 8.76
+
+
+@xs.process
+class OrographicDrainageDischarge(FlowAccumulator):
+    """Flowaccumulation including orographic precipitation
+    """
+    runoff = xs.foreign(OrographicPrecipitation, 'precip_rate')
+    discharge = xs.on_demand(
+        dims=('y','x'), description='discharge from orographic precipitation', attrs={"units": "mm/h"}
+    )
+
+    def run_step(self):
+        super().run_step()
+
+        # scale mm/h to m/yr
+        self.flowacc *= 8.76
```

Comment:
We actually want to scale from mm^3/h to m^3/yr, so the conversion factor should be 8.76e-6 instead.

That said, it makes me wonder if absolute (scaled) values of discharge are the right values to be used directly as input for computing stream power channel erosion:

- [Fastscapelib-fortran's docs](https://fastscape-lem.github.io/fastscapelib-fortran/#_fastscape_set_precip) suggests using relative values for runoff (i.e., `runoff = OrographicPrecipitation.precip_rate / OrographicPrecipitation.preci_base`), since the spatially averaged precipitation magnitude is already included implicitly in the parameters of the stream power law.

- I'm wondering whether using discharge per unit with would make more sense, as in the classic stream power law we already use drainage area expressed in m^2.

Any thoughts on this @jeanbraun ?